mailpoet/mailpoet
# Remove unused old migration code

## Description

This PR removes the code and the tests for the old SendingQueue\Migration task worker. 
It modifies tests that were using this worker.
It also updates the PHPStan baselines with references to the removed code.

## Code review notes

_N/A_

## QA notes

Only non-regression testing: activate/deactivate the plugin and make sure scheduled tasks run.

## Linked PRs

_N/A_

## Linked tickets

[MAILPOET-4478]

## After-merge notes

_N/A_


[MAILPOET-4478]: https://mailpoet.atlassian.net/browse/MAILPOET-4478?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
